### PR TITLE
Add COVID-19 Chest X-Ray Dataset

### PR DESCRIPTION
**Pull request by vmtang11**
## Add COVID-19 Chest X-Ray Dataset

* Dataset Name: covid_cxr
* Issue Reference: https://github.com/tensorflow/datasets/issues/2396
* `dataset_info.json` Gist: https://gist.github.com/vmtang11/5544e8412330e1db5623b36f450c8fa9
  
## Description

This dataset contains chest x-ray images of normal, pneumonia, and COVID-19 patients sourced from five different datasets. It currently has 507 COVID-19, 7,966 normal, and 5,468 pneumonia images in the training set and 100 COVID-19, 885 normal, and 594 pneumonia images in the test set. This dataset can be loaded in three different resolutions: 224x224, 480x480, and original varying resolutions.   
  
## Checklist
* [x] Address all TODO's
* [x] Add alphabetized import to subdirectory's `__init__.py`
* [x] Run `download_and_prepare` successfully
* [x] Add [checksums file](https://www.tensorflow.org/datasets/add_dataset#2_run_download_and_prepare_locally)
* [x] Properly cite in `BibTeX` format
* [x] Add passing test(s)
* [x] Add test data
* [x] If using additional dependencies (e.g. `scipy`), use [lazy_imports](https://www.tensorflow.org/datasets/add_dataset#extra_dependencies) (N/A)
* [x] Add data generation script (N/A)
* [x] [Lint](https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style) code

### CODE REVIEW

**Comment by vmtang11** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdatasets%2Fpull%2F2454) for more info**.

@googlebot I signed it!

**Comment by VishaalVenkatesh** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdatasets%2Fpull%2F2454) for more info**.

@googlebot I fixed it.

**Comment by VishaalVenkatesh** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdatasets%2Fpull%2F2454) for more info**.

@googlebot I consent.

**Comment by vmtang11** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Fdatasets%2Fpull%2F2454) for more info**.

@googlebot I consent.

**Comment by VishaalVenkatesh** [comment]
@googlebot I consent.

**Comment by ccl-core** [comment]
Hello @vmtang11 and @VishaalVenkatesh , thank you for your contributions!

It looks like the `_DESCRIPTION` field in both datasets is not complete, as it starts with the sentence *This dataset contains....*

Could you please complete the fields, so that the dataset documentation is completed?
